Gopkg.toml

```diff
@@ -151,5 +151,5 @@
   version = "0.8.1"
 
 [[constraint]]
-  name = "gopkg.in/gormigrate.v1"
-  version = "1.4.0"
+  name = "github.com/j16r/gormigrate"
```

Comment:
Perhaps get in touch with @se3000 and let's make a `github.com/smartcontractkit/gormigrate` branch instead of your personal one?

Reply:
Also, instead of changing the name of the import, which results in golang code changes, ie:
```
gormigrate "gopkg.in/gormigrate.v1"
```

to

```
gormigrate "github.com/j16r/gormigrate"
```

You can use the `source` key: https://golang.github.io/dep/docs/Gopkg.toml.html#source

Gopkg.toml: https://github.com/smartcontractkit/chainlink/blob/master/Gopkg.toml#L25
```
[[constraint]]
  name = "github.com/gin-gonic/gin"
  source = "github.com/smartcontractkit/gin"
  branch = "staticfs-noroute"
```

Reply:
It's a bit weird since the package name is also the version, so if we keep `gormigrate.v1` it's ever so slightly misleading. Happy to make the change though.

As for making it a smartcontractkit repo, yeah can do, but I can't migrate the PR, so it'll just be a duplicate.